.gitignore

```diff
@@ -1,6 +1,5 @@
 *-t
 *.ctest
-*.reject
```

Comment:
Why is that?

Reply:
Because it is junk and should be deleted by `git clean -f`.

Reply:
Well, from this point of view ```*.o``` are probably junk as well. As well as other generated files that are in ```.gitignore```.

I'm not completely sure here. So I'd better bring this up to other devs.

Reply:
Yes, there are some trash that is not supposed to be here (like `*.tmp` f.ex.). OTOH there are no `*.orig` which comes as a pair to `*.reject`.

Reply:
Ok, so there're two kinds of ```*.reject```: those that are created by patch and those that are generated by mysql-test-run.pl. Still, it needs some thinking.

Reply:
Just figured out that those that come from `patch` are in fact `*.rej`. I have nothing against `*.reject`. Updated commit.